docs/source/contributing/docs.rst

```diff
@@ -59,21 +59,20 @@ living document that grows over time, so feel free to add to it / change it!
 Our entire documentation does not yet fully conform to these conventions yet,
 so help in making it so would be appreciated!
 
-``pip`` invocations
--------------------
+``pip`` invocation
+------------------
 
-There are three ways to invoke a ``pip`` command:
+There are many ways to invoke a ``pip`` command, we recommend the following
+approach:
 
-1. ``pip``
-    This points to python3 pip if you are in an appropriate environment,
-    but in some cases it can actually point to a python2 pip.
+```
```

Comment:
This doesn't work in rst does it?

Reply:
Correct. Code block would be the correct approach in rst.

Reply:
damnit! sorry, brain fart...too many PRs where I'm switching back and forth between md and rst :-)